WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -900,6 +900,7 @@ extension MySiteViewController: BlogDetailsPresentationDelegate {
     }
 
     func presentBlogDetailsViewController(_ viewController: UIViewController) {
+        viewController.loadViewIfNeeded()
```

Comment:
`CATransaction` didn't work because this `handler` property in `MeViewController` was `nil`. And that property is `nil` because the `MeViewController` `viewDidLoad` method wasn't called before `self.meViewController?.navigateToAccountSettings` is called.

![CleanShot 2023-09-27 at 18 10 07@2x](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/02b0bb36-52c9-465f-abb6-8dc8f7bda6fa)